backend/lib/endpoints/users.js

```diff
@@ -37,6 +37,17 @@ async function routes(app) {
       };
     },
   );
+
+  app.put(
```

Comment:
If we're creating a new profile, wouldn't POST be more restful? Unless are we updating an existing user's profile?

Reply:
Agreed, this should be POST since it looks like we are creating a new resource, rather than updating an existing one.

Reply:
I am not sure about this as I had discussion with Robin this morning and he mentioned the following: "POST is creating a new entity so i think we already do that because we have a user
you just have to update user info"

Reply:
yeah i said to use PUT or PATCH because we update user data that already signed up, but see my previous comment it depends a bit on the specific use case how we can design it best

Reply:
@robinv85 Ah sounds good. I initially thought this was for user registration so I suggested to use POST. But PUT definitely makes more sense since the user already has a profile.

Reply:
yeah with PUT you have to send the full user object in the request body so for consistency with `updatePost` in `/posts/:postId` we could choose to always prefer PATCH 
I've used `PUT` to add comments and likes since those are very small sub resources that have separate endpoints ...